accelerator/core/serializer/serializer.c

```diff
@@ -531,21 +531,26 @@ status_t ta_send_transfer_req_deserialize(const char* const obj, ta_send_transfe
     if (raw_message) {
       msg_len = msg_len * 2;
       req->msg_len = msg_len * 3;
+      if (req->msg_len > NUM_TRITS_MESSAGE) {
```

Comment:
The `req->msg_len` should not be `msg_len * 3`.

Reply:
`req->message` is defined as `flex_trit_t message[FLEX_TRIT_SIZE_6561];` It is `flex_trit`, not `trit`.